app.py

```diff
@@ -1,24 +1,42 @@
 import os
-from urllib.request import urlopen
-# from urllib.error import URLError, HTTPError
+from urllib.request import Request, urlopen
+from urllib.error import URLError, HTTPError
 from bs4 import BeautifulSoup
 from dotenv import load_dotenv
-# import sys
 load_dotenv()
 
 
-# TODO Send Push Notification
+# TODO send push notification
 
 KINDLE = os.getenv('KINDLE')
 
 
+def isAmazonURL(url):
+    return url.find("https://www.amazon.com") != -1
+
+
 def url_to_html(url):
-    response = urlopen(url)
-    return response.read()
+    isAmazonURL = isURL(url)
+
+    if isAmazonURL:
+        try:
+            req = Request(url)
+            response = urlopen(req)
+            return response.read()
+        except HTTPError as e:
+            print(e.status, e.reason)
+        except URLError as e:
+            print(e.reason)
+
+    return False
```

Comment:
```suggestion
    if not isAmazonURL:
         return False
    try:
        req = Request(url)
        response = urlopen(req)
        return response.read()
    except HTTPError as e:
        print(e.status, e.reason)
    except URLError as e:
        print(e.reason)
```

faz um guard clause aqui (https://qvault.io/clean-code/guard-clauses/)
pra evitar ficar com muito código alinhado